npm/vue2/src/index.ts

```diff
@@ -266,6 +265,10 @@ declare global {
   }
 }
 
+const cleanup = () => {
+  Cypress.vueWrapper?.destroy()
```

Comment:
Any need to do `innerHTML = ''` for the root el here? Curious why Vue 3 does it but Vue 2 does not

Reply:
See https://github.com/cypress-io/cypress/pull/24470#discussion_r1012960512